### ISSUE TITLE
Clear search when clicking catagories

### ISSUE THREAD

**Issue by MrTinkerman** [partkeepr/PartKeepr#554]
Hi, 

A feature request from MrExplore here, posting on another account.

Request/consideration: Clear the search field when clicking a category

1. Let's say I do a search for "steppermotor"
2. I found the motor, yay :)
3. Not I would like to have a look in the category "stepperdrivers" to see what drivers I have lying around.
4. I click the category "stepperdrivers"
5. Now the category shows up empty because I did not clear the term "steppermotor" from the searchbox.
6. I get confused because I was sure I had some stepperdrivers :)

I know I could remember to clear the search field, but it seems more intuitive / obvious that I would like to see all items in a category when I click it.

Thanks

**Comment by felicitus** [comment]
Hmm I got a split opinion on that one. Sometimes users like to search for a keyword or have a filter from the filter panel active and browse through categories.

Maybe a warning message above the results indicating that a filter is active would do the trick?
![filter](https://cloud.githubusercontent.com/assets/162974/12241779/796758fa-b895-11e5-8006-53515411945c.png)

**Comment by MrTinkerman** [comment]
Yes, you do have a point there. A note/warning would be nice alternative solution.
